corehq/apps/reports/models.py

```diff
@@ -236,3 +225,18 @@ def create_jwt(self):
             }
         )
         return token
+
+
+class TableauGroup(models.Model):
+    server = models.ForeignKey(TableauServer, on_delete=models.CASCADE)
+    name = models.CharField(max_length=64)
+    group_id = models.CharField(max_length=32)
```

Comment:
Is this the primary key of the group? If so, convention would be to have it just be `id` instead of having `group.group_id`

Reply:
Got it, having it as a separate field sounds reasonable then. I think we have a couple other SQL models that are similar (an HQ-generated, auto-incrementing id that isn't used much, then a second id field that's the main primary key).